packages/components/psammead-navigation/src/DropdownNavigation/index.jsx

```diff
@@ -137,6 +154,34 @@ const MenuButton = styled.button`
     display: none;
     visibility: hidden;
   }
+  @media (min-width: ${GEL_GROUP_B_MIN_WIDTH}rem) {
```

Comment:
Just wanted to clarify, why does this media query start from 320px?

Reply:
This uses the typography groups in order to match when the line-height changes in the typography script object. These are based on the `GEL_GROUP_[A|B|C|D]` as I understand it, so I have started the media queries on those.